packages/react/src/components/Authenticator/shared/FormField.tsx

```diff
@@ -40,8 +43,9 @@ export function FormField({
           countryCodeName="country_code"
           autoComplete={autoComplete}
           hasError={hasError}
+          aria-describedby={ariaDescribedBy}
```

Comment:
I think you meant to use aria-errormessage?
https://github.com/aws-amplify/amplify-ui/issues/1737

Reply:
Update after offline discussion -- `aria-errormessage` is not well supported: [descriptiveText](https://github.com/aws-amplify/amplify-ui/issues/1737#issuecomment-1132205724) and we should use `aria-describedBy`. I could confirm this with MacOS voiceover as well.

One concern is that `aria-describedBy` is a property of `TextField` used for descriptive texts ([docs](https://ui.docs.amplify.aws/react/components/textfield)). But Authenticator doesn't use any descriptive texts, so IMO we can proceed with this change. Lmk what you think 🙏